app/models/legislator.rb

```diff
@@ -0,0 +1,93 @@
+class Legislator < ActiveRecord::Base
+  belongs_to :district
+  belongs_to :state
+  has_and_belongs_to_many :campaigns
+
+  validates :bioguide_id, presence: true, uniqueness: true
+  validates :chamber, inclusion: { in: %w(house senate) }
+  validates_presence_of :district, if: :representative?
+  validates_absence_of  :state,    if: :representative?
+  validates_presence_of :state,    if: :senator?
+  validates_absence_of  :district, if: :senator?
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.